### ISSUE TITLE
bug(live-update): `resetOnUpdate` configuration option not working

### ISSUE THREAD

**Issue by luisbytes** [capawesome-team/capacitor-plugins#256]
### Plugin(s)

- [ ] Android Foreground Service
- [ ] Android Battery Optimization
- [ ] App Update
- [ ] Background Task
- [ ] Badge
- [ ] Bluetooth Low Energy
- [ ] Cloudinary
- [ ] Datetime Picker
- [ ] File Compressor
- [ ] File Opener
- [ ] File Picker
- [X] Live Update
- [ ] Managed Configurations
- [ ] NFC
- [ ] Photo Editor
- [ ] Printer
- [ ] Screen Orientation
- [ ] Zip

### Version

6.3.0

### Platform(s)

- [X] Android
- [ ] iOS
- [ ] Web

### Current behavior

The `resetOnUpdate` configuration option does not seem to work. Regardless of the value of `resetOnUpdate`, the application is always reset to the default bundle during a native update.

### Expected behavior

The application should not be reset to the default bundle if `resetOnUpdate` is set to `false`.

### Reproduction

-

### Steps to reproduce

1. Add resetOnUpdate in false
2. Install a new native version

### Capacitor doctor

💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 6.1.2
  @capacitor/core: 6.1.2
  @capacitor/android: 6.1.2
  @capacitor/ios: 6.1.2

Installed Dependencies:

  @capacitor/ios: not installed
  @capacitor/cli: 6.1.2
  @capacitor/core: 6.1.2
  @capacitor/android: 6.1.2

### Before submitting

- [X] I have read and followed the [bug report guidelines](https://capawesome.io/contributing/bug-reports/).
- [X] I have attached links to possibly related issues and discussions.
- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by robingenz** [comment]
A reset is only performed if `resetOnUpdate` is set to `true`, see https://github.com/capawesome-team/capacitor-plugins/blob/761f65510f910ec8536a59606ae2ea816bb0d1fc/packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/LiveUpdate.java#L82

Did you run `npx cap sync` after updating the config?

**Comment by luisbytes** [comment]
> A reset is only performed if `resetOnUpdate` is set to `true`, see
> 
> https://github.com/capawesome-team/capacitor-plugins/blob/761f65510f910ec8536a59606ae2ea816bb0d1fc/packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/LiveUpdate.java#L82
> 
> Did you run `npx cap sync` after updating the config?

Yes, I put logs to print the configuration that was arriving and everything was correct, but every time I install a new apk it changes to the buildin

**Comment by robingenz** [comment]
I've just tested it and i can reproduce your issue. However, it is not the plugin that performs this reset but it seems to be Capacitor itself. I will have to take a closer look at this. Feel free to look into it yourself and create a PR.
